src/main/java/teammates/storage/entity/FeedbackResponseComment.java

```diff
@@ -36,8 +36,13 @@
     /** The foreign key to locate the FeedbackQuestion object. */
     private String feedbackQuestionId;
 
-    /** The course-specific email used by the giver of the comment. */
-    private String giverEmail;
+    /** The giver of the comment.
+     * It is email in case when comment giver is a student or instructor, and team name in case of team. */
+    private String commentGiver;
```

Comment:
Don't change entity class property name. The name will be used to persistent the entity.